archimedes_bin/main.cpp

```diff
@@ -1,14 +1,15 @@
-#include "examples/NvrhiRendererTestApp.h"
+// #include "examples/NvrhiRendererTestApp.h"
+#include "examples/PhysicsTestApp.h"
```

Comment:
This include should be moved to `examples/Examples.h` after merge/rebase with `master` branch.